index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <link rel="stylesheet" href="styles.css">
```

Comment:
- [ ] Ths CSS is not connected as the name of your file is `style.css` not `styles.css` which is why your project is not connected to the styling.

Reply:
done